src/components/DateRangePicker.jsx

```diff
@@ -529,7 +529,9 @@ class DateRangePicker extends React.Component {
 
     const { isDateRangePickerInputFocused } = this.state;
 
-    const onOutsideClick = (!withPortal && !withFullScreenPortal) ? this.onOutsideClick : undefined;
+    const enableOutsideClick = (!withPortal && !withFullScreenPortal);
+    const Wrapper = enableOutsideClick ? OutsideClickHandler : 'div';
```

Comment:
it seems like we should be able to avoid adding the wrapping div entirely by only rendering the wrapper when `enableOutsideClick` is true?

Reply:
I was thinking that I didn't want to change the DOM structure in this PR, but this is true. Do you think it could potentially cause problems if we removed this div?

Reply:
Hmm, I can't see why